packages/legend-extension-dsl-serializer/src/components/studio/BindingElementEditor.tsx

```diff
@@ -0,0 +1,450 @@
+                  modelUnitState.packageableElementExcludes.map(
+                    (elementRef) => (
+                      <ElementEditor
+                        key={elementRef.value.uuid}
+                        modelUnitState={modelUnitState}
+                        elementRef={elementRef}
+                        modelInclude={modelInclude}
+                        isReadOnly={isReadOnly}
+                      />
+                    ),
+                  )}
+
+                <div className="binding-model-editor__panel__content__form__section__list__new-item__add">
+                  <button
+                    className="binding-model-editor__panel__content__form__section__list__new-item__add-btn btn btn--dark"
+                    disabled={!allowAddingElement}
+                    tabIndex={-1}
+                    onClick={addElement}
+                    title="Add Element"
+                  >
+                    Add Value
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  },
+);
+
+const BindingGeneralEditor = observer(
+  (props: { editorState: BindingEditorState; isReadOnly: boolean }) => {
+    const { editorState, isReadOnly } = props;
+    const editorStore = useEditorStore();
+    const binding = editorState.binding;
+    const schemaSets = Array.from(
+      editorStore.graphManagerState.graph.allOwnElements,
+    ).filter(
+      (type: PackageableElement): type is SchemaSet =>
+        type instanceof SchemaSet,
+    );
+    const schemaSetOptions = schemaSets.map((e) => ({
+      value: e,
+      label: e.path,
+    }));
+    const onSchemaSetChange = (
+      val: { label: string; value: SchemaSet } | null,
+    ): void => {
+      binding.setSchemaId(undefined);
+      return binding.setSchemaSet(val?.value);
+    };
+    const selectedSchemaSet = {
+      value: binding.schemaSet,
+      label: binding.schemaSet?.path,
+    };
+    const schemaIdOptions = selectedSchemaSet.value?.schemas.map((e) => ({
+      value: e.id,
+      label: e.id,
+    }));
+    const onSchemaIdChange = (
+      val: { label: string; value: string } | null,
+    ): void => binding.setSchemaId(val?.value);
+    const selectedSchemaId = {
+      value: binding.schemaId,
+      label: binding.schemaId,
+    };
+    const projectSelectorRef = useRef<SelectComponent>(null);
+    const contentTypeOptions = Object.values(CONTENT_TYPE).map((e) => ({
+      value: e,
+      label: e,
+    }));
+    const onContentTypeChange = (val: {
+      label: CONTENT_TYPE;
+      value: CONTENT_TYPE;
+    }): void => binding.setContentType(val.value);
+    const selectedContentType = {
+      value: binding.contentType,
+      label: binding.contentType,
+    };
+    return (
+      <div className="binding-general-editor">
+        <div className="binding-general-editor__section__header__prompt">
+          Select Content Type
```

Comment:
-> `Content Type` (this is the title of the block, not the description/placeholder)